publications-2019.bib

```diff
@@ -1851,7 +1851,8 @@ @Article{2019:rudraraju.moulton.ea:computational
   month   = jul,
   doi     = {10.1371/journal.pcbi.1007213},
   publisher = {Public Library of Science ({PLoS})},
-  editor  = {Qing Nie}
+  editor  = {Qing Nie},
+  preprint = {https://arxiv.org/abs/1901.00497}
```

Comment:
I don't think there is any use in listing the preprint. The *normative* publication is the one by the publisher; preprints may or may not coincide in text with the actual publication, and often are only early versions. I do not think that we should link to it.

Separately, the editor should also not be listed.

Reply:
> Separately, the editor should also not be listed.

For journals, I agree on this. I will remove the editor in this entry.